deltacat/storage/model/push_down.py

```diff
@@ -0,0 +1,13 @@
+class RowFilter:...
```

Comment:
Can we move this file and the other scan models into dedicated `storage/model/scan/*.py` files?